src/main/java/bio/terra/cli/command/Config.java

```diff
@@ -0,0 +1,14 @@
+package bio.terra.cli.command;
+
+import bio.terra.cli.command.config.Set;
+import picocli.CommandLine.Command;
+
+/**
+ * This class corresponds to the second-level "terra config" command. This command is not valid by
+ * itself; it is just a grouping keyword for it sub-commands.
+ */
+@Command(
+    name = "config",
+    description = "Commands related to the configuration of the Terra CLI and current context.",
+    subcommands = {Set.class})
```

Comment:
nit: List.class is missing from here, but exists in the branch. I see it's just an empty placeholder class; even so, I'd either connect it in here or exclude from this PR / branch altogether.

Reply:
I hadn't pushed it, yet. Set, List, and Status are now sub-commands of the Server command. (I changed from Config -> Server because the only thing you can set so far is the server. Even workspace will be set by changing directories. I don't have strong feelings about that, so if you prefer Config I can put it back. Prefer to do so in a follow-on PR though.)